tools/provision/lib.sh

```diff
@@ -9,17 +9,17 @@
 
 function install_cmake() {
   if [[ ! -f /usr/local/bin/cmake ]]; then
-    if [[ ! -f cmake-2.8.12.2.tar.gz ]]; then
+    if [[ ! -f cmake-3.2.1.tar.gz ]]; then
```

Comment:
2.8.12.2 (the last 2.8 version) is missing a `-ltinfo`. Homebrew is using 3.2.1, we should rebuild our build host's dependencies to converge on cmake 3.2.1 across all builds.